### DISCUSSION TITLE
Read attitude files in CHAMP format

### DISCUSSION THREAD

**Discussion by ParisaShafiei** [groops-devs/groops#136] [Q&A] [answered]
Dear developers, 

For the kinematic orbit determination of a Cubesatellite mission, I have the attitude files in CHAMP format. I would like to convert them to star camera files in GROOPS. I tried with the program called "Champ2AccStar" to read the attitude files and convert them to star camera files, but the final output is empty and it seems this program is not suitable for reading the attitude files. Therefore, I wanted to ask you how we can read the attitude files in CHAMP format in GROOPS. 

Like always thank you in advance for your time and help. 

With kind regards, 
Parisa

**Comment by bsuesser-rechberger** [comment]
Hello Parisa, 

please send me the one attitude file of the cube satellite mission. I will take a look.

Best regards,
Barbara

**Reply by ParisaShafiei** [reply]
Hello Barbara, 

Thank you for your reply. I will send one on the attitude files by email to you. Thanks for the time you will spent on checking the file. 

With kind regards,
Parisa

**Comment by bsuesser-rechberger** [comment, marked as answer]
Dear Parisa, 

 the problem is, that in your file a keyword is used ('sca') which is not defined in [CHAMP Data Format](http://op.gfz-potsdam.de/champ/docs_CHAMP/CH-GFZ-FD-001.pdf). 

Best regards,  Barbara

**Reply by ParisaShafiei** [reply]
Dear Barbara, 

That is right. Thank you so much for your help. I will remove this key from each line inside the attitude file.

With kind regards, 
Parisa
